Basic-Video-Chat/README.md

```diff
@@ -28,25 +28,13 @@ the sample application, install it and these other dependencies:
 
 The OpenTok Linux SDK for x86_64 (adm64) architecture is available as a Debian
 package. For Debian we support Debian 9 (strech) and 10 (buster). We maintain
-our own Debian repository on Bintray. For Debian 10, follow these steps
+our own Debian repository on packagecloud. For Debian 10, follow these steps
```

Comment:
nit: The package manager provider/service name is a trademark (I guess) so IMO it should be `Packagecloud`. Same for the other README.md files.